model/src/main/scala/uk/gov/ons/addressIndex/model/db/index/HybridAddress.scala

```diff
@@ -63,6 +67,8 @@ object HybridAddress {
 
       HybridAddress(
         uprn = hit.sourceAsMap("uprn").toString,
+        parentUprn = hit.sourceAsMap("parentUprn").toString,
+        relatives = hit.sourceAsMap("relatives").toString,
```

Comment:
This won't work with `relatives` as it is an `Array[Map[String, Any]]`
Look at how the lpi and paf fields are analysed and it should be more or less the same for `relatives`